### PR TITLE
added simplifying comments

### PR DESCRIPTION
**Pull request by advik-student-dev**
### Describe your change:

added some comments to make the code reader able to visualise the equations (i.e write the equations in an algebraic manner so that another programmer will know the exact equation). not added comments like this to all of the calcs

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by advik-student-dev** [comment]
idk what doctests are

File: maths/area.py [hunk 1]
```diff
@@ -289,6 +293,8 @@ def area_trapezium(base1: float, base2: float, height: float) -> float:
         ...
     ValueError: area_trapezium() only accepts non-negative values
     """
+    
+    # Area = (parellel side 1 + parellel side 2) * height / 2
```
Comment (CaedenPH) [inline maths/area.py:297]:
```suggestion
    # Area = (parallel side 1 + parallel side 2) * height / 2
```

**Review by CaedenPH — COMMENTED** [review]
In my opinion, some of the comments look a bit messy, for example the cube one - It is way too long, you may want to add a line break

**Comment by advik-student-dev** [comment]
Thanks for suggestions! Have he suggestions been committed now I. The pull request?

**Review by advik-student-dev — COMMENTED** [review]
line 26 length reduced

**Pull request closed without merging**